webservices/common/util.py

```diff
@@ -16,3 +16,9 @@ def filter_query(model, query, fields, kwargs):
         column = getattr(model, field)
         query = query.filter(column.in_(value))
     return query
+
+def make_pdf_url(imagenumber):
```

Comment:
This looks like a duplicate from `webservices/utils.py`. It also doesn't work, since `self` isn't defined in this context. I think we can just delete this function.